### ISSUE TITLE
SEGAS-00008 Accessibility out of date

### ISSUE THREAD

**Issue by mikemilleresq** [UKHomeOffice/engineering-guidance-and-standards#502]
We welcome feedback on all of the content on the site. Please let us know what you think we could change or improve below

**Which content do you think should be reviewed?**
https://engineering.homeoffice.gov.uk/standards/accessibility/

**Why do you think we should review this?**
I believe https://engineering.homeoffice.gov.uk/standards/accessibility/ should match the https://design.homeoffice.gov.uk/accessibility/standard when talking about WCAG.  The SEGAS-00008 states it aligns with WCAG 2.1 but the Accessibility standard has been updated to align to WCAG 2.2

**Do you have a suggestion for how this could be improved?**
The paragraph should be updated to reference WCAG 2.2 or say alignment to the latest WCAG standard reference in the HO Standard.

**Additional information**
No

**Please confirm the below**

- [YES] I have checked through the issues on the repository and don't think this has already been suggested

**Comment by edhamiltonHO** [comment]
Have created PR #503 that changes wording to refer to latest WCAG as opposed to a specific version
